Makefile.am

```diff
@@ -259,6 +259,20 @@ test_runner_LDFLAGS = -no-install
 check-local:
 	$(AM_V_at)./test_runner
 
+lcf2xml_SOURCES = tools/lcf2xml.cpp
+lcf2xml_CPPFLAGS = $(liblcf_la_CPPFLAGS)
+lcf2xml_CXXFLAGS = $(liblcf_la_CXXFLAGS)
+lcf2xml_LDADD = liblcf.la $(liblcf_la_LIBADD)
```

Comment:
This looks like it overlinks with unneeded icu.
Also should be enough to link with lcf really, libtool should figure out the rest.

(needs testing)

Reply:
works without